torchrl/trainers/helpers/replay_buffer.py

```diff
@@ -3,9 +3,10 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-from argparse import ArgumentParser, Namespace
+from dataclasses import dataclass
 
 import torch
+from omegaconf import DictConfig
```

Comment:
same let's check if we need that

Reply:
https://github.com/BoboBananas/rl/pull/3/commits/c78c4954fa0816a8583a06938a4a7fd12c27204a